src/Controller/PayWithPayPalFormAction.php

```diff
@@ -74,7 +59,7 @@ public function __invoke(Request $request): Response
         $token = $this->authorizeClientApi->authorize($paymentMethod);
         $clientToken = $this->identityApi->generateToken($token);
 
-        return new Response($this->twig->render('@SyliusPayPalPlugin/payWithPaypal.html.twig', [
+        return new Response($this->twig->render('@SyliusPayPalPlugin/pay_with_paypal.html.twig', [
```

Comment:
At least these base templates would be good to list in the UPGRADE file that they were changed